app/models/user/UserStatTable.scala

```diff
@@ -341,12 +341,15 @@ object UserStatTable {
         |    INNER JOIN user_stat ON sidewalk_user.user_id = user_stat.user_id
         |    INNER JOIN mission ON sidewalk_user.user_id = mission.user_id
         |    INNER JOIN label ON mission.mission_id = label.mission_id
+        |    INNER JOIN audit_task ON label.audit_task_id = audit_task.audit_task_id
```

Comment:
Actually, let's punt on the Leaderboard. It's weird to only change the number of labels and not the missions, distance, or accuracy. And it's also weird to change any of them on the leaderboard, but not in the count on their user dashboard or in the accuracy rating they see on their dashboard or the Explore page.

I think that at some point we can aim to make changes to the actual `user_stat` table when we mark old data as low_quality, but that just adds another entire layer of complication. And when we do that, we can adjust the leaderboard as well.